.eslintrc

```diff
@@ -48,7 +48,8 @@
         "import/no-unresolved": 0,
         "import/extensions": 0,
         "import/no-extraneous-dependencies": 0,
-        "react/no-multi-comp": 0
+        "react/no-multi-comp": 0,
+        "react/no-array-index-key": 0
```

Comment:
I've been wanting to not require adding eslint overrides to use our story examples. I've been assuming people will cut and paste our story code and wanting to keep our examples directly useful.

Reply:
sure I will try to simplify it.